corehq/apps/users/templates/users/partials/reset_password.html

```diff
@@ -1,5 +1,2 @@
 {% load crispy_forms_tags %}
-<form id="reset-password-form" class="form-horizontal" action="{% url "change_password" domain couch_user.user_id %}" method="post">
-  {% csrf_token %}
-  {% crispy reset_password_form %}
-</form>
+{% crispy reset_password_form %}
```

Comment:
Might as well get rid of this file now that it's just two lines.

Reply:
Its used as a template for rendering ajax HTML response, so we still need it?

Reply:
aha, nevermind then

Reply:
FWIW, crispy forms can also render in python without a template:
```python
from crispy_forms.utils import render_crispy_form
render_crispy_form(form, context=context)
```

Reply:
Sweet.

Danny has not shared much about why this template was ever present in https://github.com/dimagi/commcare-hq/commit/6dfa849e2f210382454db11b86f799a15a89c6db, but does look safe to remove.

https://github.com/dimagi/commcare-hq/pull/28787/commits/78d08aead05e6a83f74122d677aa2052237e29d8